plc4x/throwing-the-towel.adoc

```diff
@@ -0,0 +1,227 @@
+
+Lately, I've simply been completely burning up.
+
+The huge pile of work, combined with the lack of reward for doing it, was simply too much for me.
+
+It's the type of burning up, where you stay up long as you've developed a sort of hate towards your bed, even if you're almost too tired to stand.
+But after giving up and giving sleep a try, you still wake up 4h later, no matter how tired you are, and you give up on trying to sleep when it's time to start working - almost being thankful that you've now got a reason to stop trying to.
+
+=== My solution
+
+So I have decided and already announced in the project, that end of March 2024 I have stopped working on the project in my free time completely if I'm not compensated for it.
+
+2 donations I got recently were a direct result of me telling the people asking for help, that this is the way they are getting support from my side and I think both are quite happy with what they got.
+
+I have set up a private repo of my own, where I'm doing work on stuff that I need or simply want to do.
+Right now it contains a completely rewritten version of an Allen Bradley Logix driver that in contrast to the PLC4X-version supports auto-discovery, browsing and reading and writing of user-defined types.
+Also will all work on the UI client for PLC4X, that I already started in that repo.
+
+Things I implement there I do plan on donating to the open-source project, however I will only do that if I receive not insignificant amount of money with donations.
+As I don't run a company I can't sell anything.
+I know this is not the way the industry operates, but to be frank: I no longer care ... if they want my stuff now they need to abide to my rules.
+
+I might continue to address things in the open-source project in my free time in the future, but these will only be because I'm using PLC4X in my home automation system or possibly someone dropped a larger donation in https://github.com/sponsors/chrisdutz/[GitHub Sponsors] or https://buymeacoffee.com/christoferu[BuyMeACoffee] accounts.
+Anything beyond my previously mentioned 4 hours per week is now history.
+
+An Apache project usually is not a one-man-show. The project will definitely continue, however at a much slower pace (as you can probably see in the following commit statistic screenshot).
+
+image::commit-activity.jpg[]
+
+=== The Future of Apache PLC4X
+
+Things might change, as me stepping back possibly makes room for people that might have been kept from participating due to my over-proportional involvement.
+If everyone knows I'm covering all the bases, why bother?
+
+If that's the case, then things will normalize again.
+
+If however this doesn't happen, there also is a chance that the opposite could happen.
+Having nobody take care of the annoying house-work, and therefore not being able to concentrate on the fun parts, could also drive people away.
```

Comment:
When there is nobody around to take care of ...
..., that could also drive people away.